### ISSUE TITLE
Reasoning currently failing on deployed version?

### ISSUE THREAD

**Issue by hlapp** [phyloref/klados#264]
If, using the currently deployed version (refs/tags/v0.3.1), I load the Brochu 2003 example and then click "Resolve against phylogenies", the reasoning seems to silently fail. At least, the phyloreference Crocodylia shows as "No nodes could be resolved" next to "Actual resolved nodes", and the same seems to be true for all other phyloreferences.

**Comment by gaurav** [comment]
Yikes, it's been so long since anybody but me has used Klados that I've just been running it locally (via `npm install` and `npm run serve`). I'll publish a new version after work today, which should update the version of Klados at https://phyloref.org/klados/. The current version of Klados has the same version of `Brochu 2003` as we used in phyx.js, so it's much shorter than this comprehensive version -- as far as I know, all of the phyloreferences in that file work correctly.

**Comment by hlapp** [comment]
I see. I think you're aware that redeployment is automatic upon every release, including I believe pre-releases. (We could delete pre-releases after the fact if you're worried about cluttering the release list.)

**Comment by gaurav** [comment]
Klados v0.4.0 now deployed to https://www.phyloref.org/klados/, and all Brochu phylorefs appear to be resolving.

**Comment by hlapp** [comment]
> all Brochu phylorefs appear to be resolving.

I can't replicate this. They continue to fail with "No nodes could be resolved". Is this a browser issue?

**Comment by gaurav** [comment]
> I can't replicate this. They continue to fail with "No nodes could be resolved". Is this a browser issue?

Nope: it worked when I tried it after deploying it two days ago, but I'm seeing the same issue as you on Firefox now. It appears to be a JSON-LD parsing error on the server end. Let me investigate and get back to you.

**Comment by gaurav** [comment]
> I can't replicate this. They continue to fail with "No nodes could be resolved". Is this a browser issue?

I've tracked it down: the JPhyloRef deployment on UFRC is working correctly, but cannot connect to the internet, which it needs to do in order to download the ontologies we include, such as CDAO (which we can work around easily) as well as downloading the JSON-LD context file (which we can also work around, but much less easier). I think UFRC might have changed their HTTP[S] proxy without telling us. I've reported this issue to UFRC, and will report back once I hear from them.
